app.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
make this change in a separate PR, right now, the review apps are using production db just to have real data, not sure if we need to tests super often new posts

Reply:
totally will! Ill revert that change